mgo.go

```diff
@@ -248,7 +248,7 @@ func (inst *MgoInstance) run() error {
 		return err
 	}
 	logger.Debugf("found mongod at: %q", mongopath)
-	if mongopath == "/usr/lib/juju/bin/mongod" {
+	if mongopath == "/usr/lib/juju/bin/mongod" || mongopath == "/usr/lib/juju/mongo3.2/bin/mongod" {
```

Comment:
This seems incredibly fragile. What happens for 3.3? Do we have to release a new version of juju?